### PR TITLE
Latest updates regarding the Roadmap

### PR DESCRIPTION
**Pull request by brunoocasali**
~Question? What we should do regarding some topics we decided to not do like `Participate to the Strapi conference in March`?~

This PR contains the latest version of the roadmap Q1, with what we did or not. 
I'll create another one with the proposals for the next one :)

### CODE REVIEW

**Comment by bidoubiwa** [comment]
I think you can just ~cross them out~.
The article about Strapi is also going to be written by @CaroFG

**Comment by curquiza** [comment]
> Question? What we should do regarding some topics we decided to not do like Participate to the Strapi conference in March?

Explain who is taking care of it now, and if it's canceled, why?
For any part we did not do, an explanation is better like `delayed in Q2`, `deprioritized`, `done by DevRel team...`

File: roadmap/2022.md [hunk 1]
```diff
@@ -22,35 +22,35 @@ Nothing is definitive: the roadmap can be adapted depending on the need of the u
...
-- [ ] Write the "Firebase + Meilisearch" article
+- [x] Finish Firebase integration
+- [ ] _(WIP, moved to Q2)_ Write the "Firebase + Meilisearch" article
 
 Strapi:
-- [ ] Make Strapi plugin compatible with the new Strapi MarketPlace. Will depend on the Strapi's docs.
-- [ ] (?) Write article "how to make your plugin compatible with the new Strapi MarketPlace"
-- [ ] Participate to the Strapi conference in March
+- [x] Make Strapi plugin compatible with the new Strapi MarketPlace. Will depend on the Strapi's docs.
+- [ ] _(will be done by dev rel team)_ Write article "how to make your plugin compatible with the new Strapi MarketPlace"
```
Comment (brunoocasali) [inline roadmap/2022.md:36]: Is that correct @bidoubiwa?

**Review by curquiza — APPROVED** [review]
💪

**Review by bidoubiwa — APPROVED** [review]
🔥🔥🔥

**Review by alallema — APPROVED** [review]
LGTM! 🎉

**Pull request merged**